### PR TITLE
Temp

### PR DESCRIPTION
**Pull request by kurasanthoshkumar**

### CODE REVIEW

File: build.txt [hunk 1]
```diff
@@ -0,0 +1,420 @@
+
+> card-issuing-frontend@0.0.0 build.e2e C:\Vikram\work space\GIT\oneplatform-recon-ui
+> gulp build.e2e --color
```
Comment (kurasanthoshkumar) [inline build.txt]: remove these lines

File: prepros.cfg [hunk 2]
```diff
@@ -0,0 +1,2469 @@
...
+          "sourceMaps": false,
+          "customOutput": false,
+          "imported": false,
+          "parents": [],
+          "type": "javascript"
+        },
+        "45bf1964": {
+          "id": "45bf1964",
+          "path": "dist/dev/app/shared/toolbar/toolbar.component.js",
+
```
Comment (kurasanthoshkumar) [inline prepros.cfg]: remove this if it is not needed

File: src/app/app.component.ts [hunk 3]
```diff
@@ -41,11 +41,11 @@ import {ProductSetupComponent} from './program/product-setup/product-setup.compo
 		name: 'ProgramAddition1',
 		component: ProgramDetailsComponent
 	},
-	// {
-	// 	path: '/ProgramList',
-	// 	name: 'ProgramList',
-	// 	component: programListComponent
-	// },
+	{
+		path: '/List',
```
Comment (kurasanthoshkumar) [inline src/app/app.component.ts]: make this  as /ProgramList

File: src/app/program/details/program-details.component.ts [hunk 4]
```diff
@@ -227,4 +227,16 @@ export class ProgramDetailsComponent implements OnInit {
     this.programForm.controls[this.binControlKey].updateValueAndValidity();
   }
 
+  Tab(index:number)
+  {
+    debugger;
```
Comment (kurasanthoshkumar) [inline src/app/program/details/program-details.component.ts]: remove this.

File: src/app/program/forms/programAddition2.component.ts [hunk 5]
```diff
@@ -5,8 +5,6 @@ import {  CORE_DIRECTIVES,
   ControlGroup,
   Validators} from '@angular/common';
 import { Program } from '../program';
```
Comment (kurasanthoshkumar) [inline src/app/program/forms/programAddition2.component.ts]: remove this file

File: src/app/program/programs.component.html [hunk 6]
```diff
@@ -12,6 +12,7 @@
 <div>
 <!-- <button (click)="addNewProgram()">Add a program</button> -->
 <button (click)="addProgramAddition1()">Program Addition 1</button>
+<button (click)="programADD()" >Program List</button>
```
Comment (kurasanthoshkumar) [inline src/app/program/programs.component.html]: method name camel case please

File: src/assets/scss/prepros.cfg [hunk 7]
```diff
@@ -0,0 +1,232 @@
+[
+  {
+    "About This File": "This is Prepros config file, https://prepros.io . Please do not edit this file, doing so can crash Prepros."
+  },
+  {
```
Comment (kurasanthoshkumar) [inline src/assets/scss/prepros.cfg]: same this if it is not needed

File: build.txt (hunk 1 above)
Comment (aditya5a2) [inline build.txt]: removed file

File: prepros.cfg (hunk 2 above)
Comment (aditya5a2) [inline prepros.cfg]: i am leaving this file as it is , because it repeats every time.

File: src/app/app.component.ts (hunk 3 above)
Comment (aditya5a2) [inline src/app/app.component.ts]: changed to programList

File: src/app/program/forms/programAddition2.component.ts (hunk 5 above)
Comment (aditya5a2) [inline src/app/program/forms/programAddition2.component.ts]: removed both html and .ts files from the source

File: src/app/program/programs.component.html (hunk 6 above)
Comment (aditya5a2) [inline src/app/program/programs.component.html]: changed to camel case

File: src/assets/scss/prepros.cfg (hunk 7 above)
Comment (aditya5a2) [inline src/assets/scss/prepros.cfg]: i am not deleting this file, leaving as it is for future conflicts

**Pull request merged**